docs/Reference/Pantheon-CLI-Syntax.md

```diff
@@ -1009,6 +1030,34 @@ receipt. Default is `false`.
     Enabling revert reason may use a significant amount of memory. We do not recommend enabling revert
     reason when connected to public Ethereum networks. 
 
+### remote-connections-limit-enabled
+
+```bash tab="Syntax"
+--remote-connections-limit-enabled[=<true|false>]
+```
+
+```bash tab="Command Line"
+--remote-connections-limit-enabled=false
+```
+
+```bash tab="Environment Variable"
+PANTHEON_REMOTE_CONNECTIONS_LIMIT_ENABLED=false
+```
+
+```bash tab="Configuration File"
+remote-connections-limit-enabled=false
+```
+
+Specify to limit the percentage of wire connections initiated by peers. Default is true. 
```

Comment:
The 'max-peers' option uses the term "P2P peer connection" where as here we use "wire connections".  Suggest making both consistent since they're referring to the same thing: "P2P connection" ("P2P peer" is redundant: peer-to-peer peer)
```suggestion
Specify to limit the percentage of P2P connections initiated by peers. Default is true. 
```

Reply:
Fixed